red/storage/index.js

```diff
@@ -95,13 +96,13 @@ var storageModuleInterface = {
         
         getLibraryEntry: function(type, path) {
             if (is_malicious(path)) {
-                return when.reject(new Error('forbidden flow name'));
+                return when.reject(new Error(log._("storage.index.forbidden-flow-name")));
```

Comment:
I changed /red/storage/index.js to return err with err.code instead of NLS'd message text.  Then, I changed /red/api/library.js to check the err.code like above.  I executed /test/red/api/library_spec.js and now no fails.  Thank you for your help!